lib/ast-converter.js

```diff
@@ -995,6 +1018,10 @@ module.exports = function(ast, extra) {
                         static: Boolean(node.flags & ts.NodeFlags.Static),
                         kind: "method"
                     });
+
+                    if (node.decorators) {
```

Comment:
In Flow, the decorators array is always present even if empty. Probably best to do that here.